doc/code_snippets/snippets/config/instances.enabled/flightrec/config.yaml

```diff
@@ -0,0 +1,16 @@
+groups:
+  group001:
+    replicasets:
+      replicaset001:
+        instances:
+          instance001:
+            iproto:
+              listen:
+              - uri: '127.0.0.1:3301'
+              flightrec:
+                enabled: true
+                logs_size: 10485800
+                logs_log_level: 5
+                metrics_period: 240
+                metrics_interval: 0.5
+                requests_size: 10485780
```

Comment:
The `flightrec` section inside `iproto` looks weird, not sure this works. I believe, it should be at the instance level (`instance001`) or the global level (both variants are OK).

Reply:
Oh, sure, that's an indentation typo. Fixed, thank you!